example/java/Android/android-build.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+PLATFORM=$1
+NDK_PATH=$2
+PACKAGE=$3
+if [ "$#" -ne 3 ]; then
+    echo "Illegal number of parameters"
+    echo "Usage: ./android-build.sh <PLATFORM> <NDK_PATH> <PACKAGE>"
+    echo "Example: ./android-build.sh arm64-v8a /Users/me/Downloads/android-ndk-r16b jp.co.soramitsu.yuna.bindings"
+    exit 1
+fi
+ANDROID_TOOLCHAIN_ARGS="-DCMAKE_SYSTEM_NAME=Android -DCMAKE_SYSTEM_VERSION=26 -DCMAKE_ANDROID_ARCH_ABI=${PLATFORM} -DANDROID_NDK=${NDK_PATH} -DCMAKE_ANDROID_STL_TYPE=c++_static"
+DEPS_DIR="$(pwd)/iroha/dependencies"
+INSTALL_ARGS="-DCMAKE_INSTALL_PREFIX=${DEPS_DIR}"
+
+# iroha develop
+git clone -b develop --depth=1 https://github.com/hyperledger/iroha
+mkdir ${DEPS_DIR}
+mkdir ${DEPS_DIR}/lib
+mkdir ${DEPS_DIR}/include
+
+# boost
+curl -OL https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.gz
+tar xf ./boost_1_66_0.tar.gz
+cp -R ./boost_1_66_0/boost ${DEPS_DIR}/include
+
+# protobuf
+git clone https://github.com/google/protobuf
+(cd ./protobuf ; git checkout b5fbb742af122b565925987e65c08957739976a7)
+cmake -Dprotobuf_BUILD_TESTS=OFF -H./protobuf/cmake -B./protobuf/host_build # build for host to get js_embed
+cmake --build ./protobuf/host_build
+sed -i.bak "s~COMMAND js_embed~COMMAND $(pwd)/protobuf/host_build/js_embed~" ./protobuf/cmake/libprotoc.cmake
+LDFLAGS="-llog -landroid" cmake ${ANDROID_TOOLCHAIN_ARGS} ${INSTALL_ARGS} -Dprotobuf_BUILD_TESTS=OFF -H./protobuf/cmake -B./protobuf/.build
+cmake --build ./protobuf/.build --target install
```

Comment:
It could be useful to add `VERBOSE=1` before cmake build invoke to track which commands are used to compile object files, libraries, and binaries, so that it is easy to track any build errors.